src/transformState.js

```diff
@@ -4,8 +4,36 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
-function transformState(state, actions) {
```

Comment:
You can place here a bunch of helper constants, for example:
```const CLEAR = 'clear';```
And use it instead of magic strings (a considered programming antipattern) in your code